products/BellHybrid/services/evtmgr/EventManager.cpp

```diff
@@ -119,6 +108,16 @@ void EventManager::initProductEvents()
         auto msg = std::make_shared<sevm::LatchStatusResponse>(latchStatus);
         return msg;
     });
+
+    connect(service::AudioPlaybackStartNotification(), [&](sys::Message *msgl) {
+        temperatureProvider.audioStarted();
+        return sys::msgHandled();
+    });
+
+    connect(service::AudioPlaybackStopNotification(), [&](sys::Message *msgl) {
+        temperatureProvider.audioStoped();
+        return sys::msgHandled();
+    });
```

Comment:
In my opinion dependency to ServiceAudio is wrong here. More general solution will be to use our PowerManager to obtain current cpu load and cpu frequency to provide useful input to temperature compensation. Was the concept for this audio playback compensation discussed before in Harmony team?

Reply:
I agree with Wojtek. The same goes for the audio service which now requires event manager dependency. As far as I know, it was found audio service is the main source of the temperature rise. Of course, the solution proposed by Wojtek is much better.